hygen/_templates/layer/new/prompt.js

```diff
@@ -5,7 +5,7 @@ const { promptArgs, readFile, getFiles } = require('../../promptUtils');
 
 const VIEWS_DIR = 'components/views'
 
-const SOURCE_TYPES = ['sql', 'bq'];
+const SOURCE_TYPES = ['sql', 'bigquery'];
```

Comment:
IMO, if we have defined a type in the lib, we should export it and use them here, instead of a duplicated string

Reply:
You are right, didn't want to make this little 'breaking-change', going to add this then